### PR TITLE
merge everything

### PR DESCRIPTION
**Pull request by azizova-leyla**
Merge latest civiform-deploy here.
Move aws related configs.
Update actions to deploy aws and azure separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
-# civiform-deploy
-Use this repo to store standardized Terraform deployment configurations for CiviForm.
+Setup for Civiform Staging Environments. See more documentation [here](https://docs.civiform.us/it-manual/sre-playbook/terraform-deploy-system).
```
Comment (nbeloglazov) [inline README.md:1]: Should we list urls to staging instances here as well? And I would clarify that this is contains configs for 2 existing staging environments. That this is not a template repo unlike civiform-deploy.

Comment (nbeloglazov) [inline README.md:1]: Can we have a command that you can run that will update bin/* scripts from `civiform-deploy`? As going forward we'll have to do it from time to time and having one-line command available here would be useful.

**Review by nbeloglazov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (azizova-leyla) [reply README.md:1]: Let me take this as todo. I did a manual cp this time, agree that it would be great to have a one command sync.

Comment (nbeloglazov) [reply README.md:1]: If it's a few cp commands - I think adding them as it is in README fine too.

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**